bindings/c/include/opendal.h

```diff
@@ -121,7 +121,6 @@ typedef struct BlockingLister BlockingLister;
  * Some services like s3, gcs doesn't have native blocking supports, we can use [`layers::BlockingLayer`]
  * to wrap the async operator to make it blocking.
  *
- * ```rust
```

Comment:
Do we miss a code block beginning here?

Reply:
It's generated from [here](https://github.com/apache/opendal/pull/4061/files#diff-5a52598b6d997d9668254e2164bfa2132b79365387edc15f7df27232d1d69b6aR62-R63). IIRC there is an issue o decouple these docs from Rust's one.

Reply:
Thus unrelated to this patch or we should not resolve it here.